test/server/media-controller.spec.js

```diff
@@ -64,11 +64,11 @@ describe( 'Testing request-filtering-agent', function() {
         request( app )
             .get( requestMetaURL  )
             .set( 'Referer', 'https://google.com?print=true' )
-            .expect( 500, /Socket is closed/ )
+            .expect( 500, /DNS lookup .* is not allowed. Because, It is private IP address/ )
```

Comment:
This change, and the one in `for a private IP address WITHOUT a Referer with allowPrivateIPAddress=false and allowMetaIPAddress=true`, assumes that:

- The intent of the test is that the request fails
- The previous message is related to different internal Node networking changes
- Testing the message is still desired (is it?)